Source/LinqToDB/Internal/DataProvider/PostgreSQL/PostgreSQLDataProvider.cs

```diff
@@ -98,6 +99,9 @@ static DateTimeOffset ConvertDateTimeToDateTimeOffset(DateTime dateTime)
 
 		protected override IMemberTranslator CreateMemberTranslator()
 		{
+			if (Version >= PostgreSQLVersion.v13)
+				return new PostgreSQL13MemberTranslator();
+
 			return new PostgreSQLMemberTranslator();
```

Comment:
feel free to alter style
```suggestion
			return Version switch
			{
				>= PostgreSQLVersion.v13 => new PostgreSQL13MemberTranslator(),
				_ => new PostgreSQLMemberTranslator(),
			};
```